dev/README.md

```diff
@@ -147,7 +147,7 @@ remedy this. By default, the setup script will use `/usr/bin/helm`.
 
 ## Different Script Folders
 
-Our CIs, dapper/drone and github actions, use a different set of scripts. CI
+Our CIs and github actions, use a different set of scripts. CI
```

Comment:
nit: maybe that's just me, but I don't understand what that comma does there :thinking: 
```suggestion
Our CIs and github actions use a different set of scripts. CI
```